lms/djangoapps/verify_student/views.py

```diff
@@ -936,7 +1028,19 @@ def results_callback(request):
         course_id = attempt.window.course_id
         course_enrollment = CourseEnrollment.get_or_create_enrollment(attempt.user, course_id)
         course_enrollment.emit_event(EVENT_NAME_USER_REVERIFICATION_REVIEWED_BY_SOFTWARESECURE)
+
     VerificationStatus.add_status_from_checkpoints(checkpoints=checkpoints, user=attempt.user, status=status)
```

Comment:
Please hide this entire block behind the `InCourseReverificationConfiguration` feature flag.  On the small chance that something unexpected goes wrong here, we need to be able to disable this code.